libsolidity/ast/AST.h

```diff
@@ -1150,6 +1151,23 @@ class TupleExpression: public Expression
 	std::vector<ASTPointer<Expression>> m_components;
 };
 
+class InlineArrayExpression: public Expression
```

Comment:
on it. Trying to figure out how exactly this should fit. Lots of moving parts here that I'm not too familiar with :)